### ISSUE TITLE
Strange call of np.linspace

### ISSUE THREAD

**Issue by mcepl** [Blosc/bloscpack#98]
https://github.com/Blosc/bloscpack/blob/45038d327258b5530d50a7a31d05b325e9b77155/bloscpack/testutil.py#L39

The third parameter of `np.linspace` is:
```
numint, optional

    Number of samples to generate. Default is 50. Must be non-negative.
```
Are you certain you want 2,000,000 samples? Besides my numpy (version 1.19.1) complains that `TypeError: 'float' object cannot be interpreted as an integer` for this parameter.

**Comment by esc** [comment]
Thanks for reporting this, the last release of Bloscpack was in 2018, so it may well be that Numpy has changed by now.

**Comment by esc** [comment]
I think the intention is to test different ways of compressing the data, hence a sufficiently large array must be provided.

**Comment by esc** [comment]
Fixed with #100
